src/main/java/org/sonarlint/intellij/config/project/SonarLintProjectBindPanel.java

```diff
@@ -160,7 +160,7 @@ private Map<String, SonarProjectDto> downloadProjectList(ServerConnection select
     try {
       return ProgressManager.getInstance().run(downloadTask);
     } catch (Exception e) {
-      var msg = e.getMessage() != null ? e.getMessage() : "Failed to download list of projects";
+      var msg = "Failed to download list of projects. Make sure an authorized connection is present.";
```

Comment:
This is not always the root cause, it could be anything (no connection, wrong token, wrong permission, etc etc).

Maybe we could log the full stack trace in the console and propose the user to check it